lib/src/typer.dart

```diff
@@ -25,23 +31,32 @@ class TyperAnimatedText extends AnimatedText {
           duration: speed * text.characters.length,
         );
 
-  Animation<int> _typingText;
+  Animation<double> _typingText;
 
   @override
   Duration get remaining => speed * (textCharacters.length - _typingText.value);
 
   @override
   void initAnimation(AnimationController controller) {
-    _typingText = StepTween(
-      begin: 0,
-      end: textCharacters.length,
+    _typingText = CurveTween(
+      curve: curve,
     ).animate(controller);
   }
 
   /// Widget showing partial text, up to [count] characters
   @override
   Widget animatedBuilder(BuildContext context, Widget child) {
-    final count = _typingText.value;
+    /// Output of CurveTween is in the range [0, 1] for majority of the curves.
+    /// It is converted to [0, textCharacters.length].
+    var count = (_typingText.value * textCharacters.length).round();
```

Comment:
`final` is better than `var` because this value is assigned _once_ and only _once_.

(Note that the following `count` adjustments do not apply to `Typer`.  It doesn't offer the extra cursor flashing as `Typewriter`.)

Reply:
Yes, `final` is the better keyword here. It keeps up with the uniformity in code as well.